lighthouse-core/gather/drivers/driver.js

```diff
@@ -489,11 +489,22 @@ class Driver {
     });
   }
 
-  beginEmulation() {
-    return Promise.all([
-      emulation.enableNexus5X(this),
-      emulation.enableNetworkThrottling(this)
-    ]);
+  beginEmulation(flags) {
+    let emulations = [];
+
+    if (!flags.disableDeviceEmulation) {
+      emulations.push(emulation.enableNexus5X(this));
+    }
+
+    if (!flags.disableNetworkThrottling) {
+      emulations.push(emulation.enableNetworkThrottling(this));
+    }
+
+    if (!flags.disableCpuThrottling) {
```

Comment:
Let's make the cpu throttle disabled by default to start. Once the other stuff like #784 is all done we can flip that flag.